cmd/operator/main.go

```diff
@@ -113,6 +113,7 @@ func main() {
 		payload.KubeProxyPath,
 		payload.IgnoreWgetPowerShellPath,
 		payload.WmcbPath,
+		payload.WICDPath,
```

Comment:
This should be part of the last commit, the variable does not exist at this point.